scripts/jointLklDM.C

```diff
@@ -517,7 +609,7 @@ void jointLklDM(TString configFileName="$GLIKESYS/rcfiles/jointLklDM.rc",Int_t s
 	  {
 	    Iact1dUnbinnedLkl* fullLkl = dynamic_cast<Iact1dUnbinnedLkl*>(sample[isample]);
 	    
-	    if(imass==0)
+	    if(!Init_canvas_samples)
```

Comment:
Can you comment on this please?

Reply:
I had to change that because now I have implemented the "skip DM mass" functionnality that allows to run a rcfile for MAGIC at a mass of 10 GeV without crashing as previously (the goal is to create a glory duck file for all masses, including the one where we are not sensitive). The counterpart is that at 10 GeV nothing will produce signal in MAGIC so we will never reach that point of the code ( "if(imass==0)" ) and for the first mass for which we will reach that point of the code the condition "if(imass==0)" will not be true anymore so the canvas won't be initialize and we won't get any plot (or the program will crash I don't remember now). This change allow everything to work nicely, by just changing this variable from False to True after initialization.